designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/ScenarioStatistics.scala

```diff
@@ -115,16 +116,51 @@ object ScenarioStatistics {
     }
   }
 
-  def getComponentStatistic(componentList: List[component.ComponentListElement]): Map[String, String] = {
+  def getComponentStatistic(
+      componentList: List[component.ComponentListElement],
+      components: List[ComponentDefinitionWithImplementation]
+  ): Map[String, String] = {
     if (componentList.isEmpty) {
       Map.empty
     } else {
-      val withoutFragments      = componentList.filterNot(comp => comp.componentType == ComponentType.Fragment)
-      val componentsByNameCount = withoutFragments.groupBy(_.name).size
+      val builtinNames = List("choice", "filter", "record-variable", "split", "variable")
```

Comment:
Can we take these names from components-api e.g. `BuiltInComponentId.Filter.name` and place it as static var?